src/include/daos_srv/daos_server.h

```diff
@@ -464,7 +464,7 @@ enum dss_xs_type {
 
 int dss_parameters_set(unsigned int key_id, uint64_t value);
 
-typedef ABT_pool (*dss_abt_pool_choose_cb_t)(crt_rpc_t *rpc, ABT_pool *pools);
+typedef ABT_pool (*dss_abt_pool_choose_cb_t)(crt_rpc_t *rpc, ABT_pool * pools);
```

Comment:
(style)  do not add new typedefs
(style)  space prohibited between function name and open parenthesis '('